docs/reference/search/async-search.asciidoc

```diff
@@ -0,0 +1,210 @@
+nor search requests that only include the  <<search-suggesters,suggest section>>.
+{ccs} is supported only with <<ccs-min-roundtrips,`ccs_minimize_roundtrips`>>
+set to `false`.
+
+[[get-async-search]]
+==== Get async search
+
+The get async search API retrieves the results of a previously submitted
+async search request given its id. The access to the results of a specific
+async search is restricted to the user that submitted it in the first place.
+
+[source,console,id=get-async-search-date-histogram-example]
+--------------------------------------------------
+GET /_async_search/FmRldE8zREVEUzA2ZVpUeGs2ejJFUFEaMkZ5QTVrSTZSaVN3WlNFVmtlWHJsdzoxMDc=
+--------------------------------------------------
+// TEST[continued s/FmRldE8zREVEUzA2ZVpUeGs2ejJFUFEaMkZ5QTVrSTZSaVN3WlNFVmtlWHJsdzoxMDc=/\${body.id}/]
+
+[source,console-result]
+--------------------------------------------------
+{
+  "id" : "FmRldE8zREVEUzA2ZVpUeGs2ejJFUFEaMkZ5QTVrSTZSaVN3WlNFVmtlWHJsdzoxMDc=",
+  "version" : 2, <1>
+  "is_partial" : true,
+  "is_running" : true,
+  "start_time_in_millis" : 1583945890986,
+  "expiration_time_in_millis" : 1584377890986,
+  "response" : {
+    "took" : 12144,
+    "timed_out" : false,
+    "num_reduce_phases" : 38,
+    "_shards" : {
+      "total" : 562,
+      "successful" : 188,
+      "skipped" : 0,
+      "failed" : 0
+    },
+    "hits" : {
+      "total" : {
+        "value" : 456433,
+        "relation" : "eq"
+      },
+      "max_score" : null,
+      "hits" : [ ]
+    },
+    "aggregations" : { <2>
+      "sale_date" :  {
+        "buckets" : []
+      }
+    }
+  }
+}
+--------------------------------------------------
+// TESTRESPONSE[s/FmRldE8zREVEUzA2ZVpUeGs2ejJFUFEaMkZ5QTVrSTZSaVN3WlNFVmtlWHJsdzoxMDc=/$body.id/]
+// TESTRESPONSE[s/"is_partial" : true/"is_partial" : false/]
+// TESTRESPONSE[s/"is_running" : true/"is_running" : false/]
+// TESTRESPONSE[s/1583945890986/$body.start_time_in_millis/]
+// TESTRESPONSE[s/1584377890986/$body.expiration_time_in_millis/]
+// TESTRESPONSE[s/"took" : 12144/"took": $body.response.took/]
+// TESTRESPONSE[s/"total" : 562/"total": $body.response._shards.total/]
+// TESTRESPONSE[s/"successful" : 188/"successful": $body.response._shards.successful/]
+// TESTRESPONSE[s/"value" : 456433/"value": $body.response.hits.total.value/]
+// TESTRESPONSE[s/"buckets" : \[\]/"buckets": $body.response.aggregations.sale_date.buckets/]
+// TESTRESPONSE[s/"num_reduce_phases" : 38,//]
+
+<1> The returned `version` is useful to identify whether the response contains
+additional results compared to previously obtained responses. If the version
+stays the same, no new results have become available, otherwise a higher version
+number indicates that more shards have completed their execution of the query
+and their partial results are also included in the response.
+<2> Partial aggregations results, coming from the shards that have already
+completed the execution of the query.
+
```

Comment:
Can you add  an explanation for `expiration_time_in_millis`, `is_partial` and `is_running` ? The last two are important since they determine how to interpret the search response (partial or not) and if the search is still running.